.changeset/shaggy-moons-peel.md

```diff
@@ -0,0 +1,5 @@
+---
+"astro": minor
+---
+
+Adds experimental support for the Actions API. Actions let you define type-safe endpoints you can query from client components with progressive enhancement built in.
```

Comment:
This is perfectly fine as is, @bholmesdev , but if you want, you can compare to Ema's experimental rerouting changeset. We *do* have a precedent of making hefty minor changesets for experimental flag: at least showing the config option but also optionally stuffing some examples in.

If you'd *like* to make this bigger, faster, stronger, you are welcome to do so!